src/modifiers.rs

```diff
@@ -195,3 +192,26 @@ impl Modifier<Response> for RedirectRaw {
         res.headers.set(headers::Location(path));
     }
 }
+
+fn mime_for_path(path: &Path) -> Mime {
+    guess_mime_type_opt(path)
+        .unwrap_or_else(|| Mime(TopLevel::Text, SubLevel::Plain, vec![]))
```

Comment:
I kept the fallback the same as before: https://github.com/conduit-rust/mime-types/blob/master/src/lib.rs#L48

mime_guess would fall back to `application/octet-stream` instead, which might be the better default. But not sure if you wanna change that in Iron.